release-notes/opensearch.release-notes-3.0.0-alpha1.md

```diff
@@ -0,0 +1,110 @@
+## 2025-02-27 Version 3.0.0-alpha1 Release Notes
+
+## [3.0.0-alpha1]
+### Breaking Changes in 3.0.0-alpha1
+- Upgrade to Lucene 10 - PR ([#16366](https://github.com/opensearch-project/OpenSearch/pull/16366))
```

Comment:
Looks like most or all of these in this section are duplicated in entries below?

Reply:
Ack removing duplicates.

Reply:
Done